client/ngapp/scripts/controllers/todo.js

```diff
@@ -137,23 +145,18 @@ angular.module('loopbackExampleFullStackApp')
   });
 
   $scope.resolveUsingSource = function(conflict) {
-    conflict.resolve(refreshConflicts);
+    conflict.resolveUsingSource(refreshConflicts);
   };
 
   $scope.resolveUsingTarget = function(conflict) {
-    if(conflict.targetChange.type() === 'delete') {
-      conflict.SourceModel.deleteById(conflict.modelId, refreshConflicts);
-    } else {
-      var m = new conflict.SourceModel(conflict.target);
-      m.save(refreshConflicts);
-    }
+    conflict.resolveUsingTarget(refreshConflicts);
   };
 
   $scope.resolveManually = function(conflict) {
-    conflict.manual.save(function(err) {
-      if(err) return errorCallback(err);
-      conflict.resolve(refreshConflicts);
-    });
+    conflict.resolveManually({
+      completed: conflict.manual.completed,
+      title: conflict.manual.title
+    }, refreshConflicts);
```

Comment:
I believe this can be simplified to `conflict.resolveManually(conflict.manual, refreshConflicts)`